apps/web/vibes/soul/examples/pages/error/index.tsx

```diff
@@ -67,7 +67,7 @@ export default function Preview() {
         searchHref="#"
       />
 
-      <Error cta={{ label: 'Try again', href: '#' }} />
+      <Error ctaAction={() => null} />
```

Comment:
Let's not pass anything. This is causing the build to fail. Another option would be to pass in 

```ts
async () => {
  'use server'

  // do nothing
}
```